spec/integration/init_web_spec.cr

```diff
@@ -28,7 +28,7 @@ describe "Initializing a new web project" do
   it "creates a full web app with generator" do
     puts "Web app generators: Running integration spec. This might take awhile...".colorize(:yellow)
     with_project_cleanup do
-      should_run_successfully "crystal run src/lucky.cr -- init.custom test-project"
+      should_run_successfully "crystal run src/lucky.cr -- init.custom test-project --no-auth"
```

Comment:
We test `gen.model User` which always fails because we were generating auth. Alternatively, we could keep generating auth, but test generating a different model.